<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Routes</class>
 <widget class="QWidget" name="Routes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="routesTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>581</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>5</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>180</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>5</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>23</number>
   </attribute>
   <column>
    <property name="text">
     <string>Широта</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Долгота</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Высота</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addPointButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить
точку</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Сбросить</string>
   </property>
  </widget>
  <widget class="QLabel" name="latitudeLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Широта</string>
   </property>
  </widget>
  <widget class="QLabel" name="longitudeLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Долгота</string>
   </property>
  </widget>
  <widget class="QLabel" name="altitudeLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Высота</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setRouteButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Установить
маршрут</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="latitudeSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.000001000000000</double>
   </property>
   <property name="value">
    <double>55.766179999999999</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="longitudeSpinBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.000001000000000</double>
   </property>
   <property name="value">
    <double>37.685009999999998</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="altitudeSpinBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QGVMap" name="mapWidget" native="true">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>511</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGVMap</class>
   <extends>QWidget</extends>
   <header>QGeoView/QGVMap.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
